ethanschreur/kids-krafts
# Customers can view a landing page

**What does this PR do?**
- This pull request allows customers to view a landing page

**Description of the task to be completed** 
- Make the customer side navigation bar with links to Home, About, Contact, and Shop
- Display craft examples at the top of the website
- Explain how the business works in a "How It Works" section
- Have a social proof section

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/
- View the "craft examples" section
- View the "how it works" section
- View the "social proof" section
- Make sure the navigation bar has links to Home, About, Contact, and Shop
